modules/nf-commons/src/main/nextflow/plugin/PluginsFacade.groovy

```diff
@@ -95,6 +98,28 @@ class PluginsFacade implements PluginStateListener {
         }
     }
 
+    protected String getPluginsIndexUrl() {
+        final url = env.get('NXF_PLUGINS_INDEX_URL')
+        if( url ) {
```

Comment:
Ideally there should not be any warning if the `url` is the current (github) index or the uri host is `*.seqera.io`